Bug: @opentui/react runtime-plugin-support.ts imports non-existent path "../src/index.js"

## Bug Description

In `@opentui/react` version 0.1.102, the file `scripts/runtime-plugin-support.ts` imports from a non-existent path.

## Steps to Reproduce

1. Install `@opentui/react@0.1.102`
2. Try to run any project using `@opentui/react`
3. Error: `Cannot find module '../src/index.js' from '...node_modules/@opentui/react/scripts/runtime-plugin-support.ts'`

## Root Cause

File: `packages/react/scripts/runtime-plugin-support.ts` line 7

```typescript
import * as opentuiReactRuntime from "../src/index.js"  // ❌ File does not exist
```

The `packages/react/src/` directory only contains `.d.ts` TypeScript declaration files, not compiled `.js` files.

## Fix

Change line 7 to:

```typescript
import * as opentuiReactRuntime from "../index.js"  // ✅ Correct path
```

The root `index.js` file exists and is the correct entry point.

## Environment

- Bun v1.3.11 (macOS arm64)
- @opentui/react: 0.1.102
- Node.js / Bun runtime

--- a/packages/react/index.ts
+++ b/packages/react/index.ts
@@ -1,0 +1,1 @@
+export * from "./src/index.js"

--- a/packages/react/scripts/runtime-plugin-support.ts
+++ b/packages/react/scripts/runtime-plugin-support.ts
@@ -4,7 +4,7 @@ import { createRuntimePlugin, type RuntimeModuleEntry } from "@opentui/core/runt
 import * as reactRuntime from "react"
 import * as reactJsxRuntime from "react/jsx-runtime"
 import * as reactJsxDevRuntime from "react/jsx-dev-runtime"
-import * as opentuiReactRuntime from "../src/index.js"
+import * as opentuiReactRuntime from "../index.js"
 
 const runtimePluginSupportInstalledKey = "__opentuiReactRuntimePluginSupportInstalled__"
 

